### ISSUE TITLE
piano touch not working 

### ISSUE THREAD

**Issue by odaymusic** [AaronDavidNewman/Smoosic#73]
hi my frind,, i see bug in ur editor look this https://prnt.sc/QtvqI5iJyAQ2

**Comment by AaronDavidNewman** [comment]
Thanks for reporting this, Oday!  It is fixed now!

**Comment by odaymusic** [comment]
yes now is working But when the piano is closed, it cannot be shown againhttps://prnt.sc/23F1_B-kNSAZ

**Comment by AaronDavidNewman** [comment]
It's now a score setting.  Score/Global Settings/Show piano widget.  Because for large scores, I like it to stay hidden due to the space it takes.

![image](https://user-images.githubusercontent.com/5438280/176980562-167501c5-7b34-49f7-a62d-a82efdba9471.png)

**Comment by odaymusic** [comment]
good idea <3
